src/cli/requirements.txt

```diff
@@ -1,5 +1,6 @@
 msal==1.20.0
-requests~=2.27.1
+# install requests 2.31.0 to fix CVE-2023-32681
+requests==2.31.0
```

Comment:
Do we want this to be a min bound?

Reply:
yeah good point

Reply:
@Porges - updated to be the minimum version allowed now